tests/CppWinRTAuthoringTests.cpp

```diff
@@ -7,8 +7,10 @@
 #if _MSVC_LANG >= 201703L
 #include <winrt/Windows.Foundation.h>
 #include <winrt/Windows.System.h>
+#include <winrt/Windows.UI.Xaml.h>
 #include <winrt/Windows.UI.Xaml.Data.h>
 #include <winrt/Windows.UI.Xaml.Input.h>
+#include <winrt/Windows.UI.Xaml.Interop.h>

```

Comment:
I don't think WIL has a WinAppSDK dependency to test against.  This coverage is limited to System XAML types.

I think there are already some WinAppSDK helpers in WIL that are similarly impossible to cover with tests.  That's unfortunate.